AutoGear.lua

```diff
@@ -2459,6 +2461,8 @@ AutoGearFrame:SetScript("OnEvent", function (this, event, arg1, arg2, arg3, arg4
 			dataAvailable = 1
 			AutoGearFrame:UnregisterEvent(event)
 		end
+	elseif event == "BAG_UPDATE_DELAYED" then
+		AutoGearConsiderAllItems()
```

Comment:
We do not want this behavior.  It would prevent removing an item manually.  AutoGear would fight against the player immediately instead of allowing the player to move their items manually as they would wish.  AutoGear should only automatically equip an item under these circumstances:

- the player's character receives a gear item for the first time from outside their inventory, like from a loot roll, vendor, or a quest reward
- the player's character's gear evaluation situation changes, like when changing specialization
- the player has manually initiated a scan

Reply:
There is a useful programming tool built into the WoW interface which can listen for events, accessible with the chat command `/eventtrace`.  If you try using that, then getting a new item, you may see the necessary event.

Currently, before any of these changes, AutoGear reads chat messages and uses a specific filter to find the item link in the chat message.  It seems likely that if Blizzard changed those chat messages, the filter may no longer match, so my advice is to start there, after confirming that the chat message event is at least firing.

Reply:
Yeah, I stumbled upon /etrace which is how I found the bag update events. I did also notice the chat parsing logic today when looking over things again. I plan to look at that further and see if I can figure out what broke there to minimize the overall changes to AutoGear.

Reply:
In case this helps you see something I am missing with less time involved than installing the game. If not no biggie, I tend to be verbose and over share in case it might be helpful.

![Screenshot 2022-12-12 at 8 32 10 PM](https://user-images.githubusercontent.com/39010826/207204230-f59f6305-9676-409f-991f-8727a28b3417.png)

Reply:
The issue seems to be the event passes the unit name with -Server appended, UnitName("Player") does not causing the listener for chat messages to break out early. Looking for the best fix for that now.